syntax = "proto3";

package metadata_processor;

service MetadataProcessor {
    rpc ProcessTrainingMetadata (stream TrainingMetadataRequest) returns (TrainingMetadataResponse) {}
}

message TrainingMetadataRequest {
    int32 training_id = 1;
    string data = 2;
}

message TrainingMetadataResponse {
}